love/src/components/Layout/Layout.jsx

```diff
@@ -238,6 +241,16 @@ class Layout extends Component {
     });
   };
 
+  checkEfdStatus = () => {
+    const url = this.props.efdConfigFile.urlStatus;
```

Comment:
I think this is missing a validation. What happens if the urlStatus is not present? It doesn't throw any error?